pkg/configtx/orderer.go

```diff
@@ -283,18 +283,18 @@ func (c *ConfigTx) AddOrdererEndpoint(orgName string, endpoint Address) error {
 
 // RemoveOrdererEndpoint removes an orderer's endpoint from an existing channel config transaction.
 // The removed endpoint and org it belongs to must both already exist.
+// Specifying an endpoint or organization name that does not exist in the orderer ConfigGroup of
+// the channel config will not return a error.
+// Removal will panic if orderer group does not exist.
```

Comment:
Will also panic if the orderer org group does not exist on line 294 when attempting to access Values